views/singleArticle/SingleArticle.jsx

```diff
@@ -62,6 +66,24 @@ function SingleArticle() {
     setViewComments((curr) => !curr)
   }
 
+  const newCommentHandler = (e) => {
+    setNewComment(e.target.value)
+  }
+
+  const submitCommentHandler = (e) => {
+    e.preventDefault()
+    const commentBody = e.target[0].value
+    if(!commentBody) alert("Whoops! You need to write your comment")
+    if(commentBody.length < 5) alert("Your comment does not meet the minimum length")
+    else {
+      const comment = {body: commentBody, author: loggedInUser.username}
+      postNewComment(article_id, comment)
+        .then((response) => {
+          console.log(response);
+        })
+    }
+  }
+
```

Comment:
Very good. You may want to use a value property in your input tag to clear the form on submit.